.github/workflows/scripts/check_for_changed_pages.js

```diff
@@ -0,0 +1,63 @@
+module.exports = {
+  getChangedPages: ({ github, context, buildDir }) => {
+    const fs = require('fs');
+    const cheerio = require('cheerio');
+    
+    const urlList = [];
+    
+    const {
+      issue: { number: issue_number },
+      repo: { owner, repo }
+    } = context;
+
+    // Use the Github API to query for the list of files from the PR
+    return github
+      .paginate(
+        'GET /repos/{owner}/{repo}/pulls/{pull_number}/files',
+        { owner, repo, pull_number: issue_number },
+        (response) => response.data.filter((file) => (file.status === 'modified' || file.status === 'added'))
```

Comment:
Could we use `async/await` here? I know one of our utilities function started using `await`, but I know we still have other scripts using the promises chaining

example of async and await from one of our scripts: 

https://github.com/aws-amplify/docs/blob/19117c75f49dcf8a81bb4c967c1ee868efa3d388/.github/workflows/scripts/utilities.js#L14-L22

Reply:
I updated this in latest commit to use async/await instead (and some formatting/comment updates)